### DISCUSSION TITLE
Showing Detail of Running Asus Service

### DISCUSSION THREAD

**Discussion by TryBugger** [seerge/g-helper#2129] [Ideas]
hello, I'm interested in this application and have recently been using this. compared to Armoury Crate, this is much better.

About the idea of showing details of running Asus service, what I think is that if this feature is available, it can give us info about what kind of Asus service is running in the background and its purpose it. and maybe also inform us that the service provided by the Asus system might already done in G-Helper.

Thank you again, appreciate ur project so much.

**Comment by Kikimahe** [comment]
Hi,

All ASUS services can be found by running `services.msc`.
There is a description of what they do.

If you have uninstalled AC, most of the remaining services are MY ASUS services.

Note that these services are installed and started every time you install an ASUS driver! And this even if you have never installed MY ASUS! :'(
Same thing for  'Armory Crate Live update' service

**Reply by TryBugger** [reply]
ah i see, i don't know we can check it from there. thanks for the information. but maybe for easier (and maybe with better visuals), is it suitable if we show it too in our tool like G-Helper for easier access? just want to ask and thanks for the information and help

**Comment by seerge** [comment]
@TryBugger you can see counter of running AC services under ``Extra``. Most of them are needed only for MyASUS only.

**Comment by Zaat** [comment]
I have all ASUS/MyASUS/Armoury Crate services stopped+disabled (they're still installed) but it shows that 1 service is still running.  I can't figure out what it is.  Is there a list of what ASUS services GHelper monitors?  Thanks.

**Reply by seerge** [reply]
You can just open windows Services app and check which specific service refuses to stop, and try to stop it there if you want.

**Reply by Zaat** [reply]
Thanks. I did look through the services and tried searching for anything ASUS or Armoury Crate related and they’re all stopped+disabled. I cannot determine which “1 running service” GHelper is referring to. It probably has a name that’s not immediately obvious it is an ASUS service, like Aura, Visual Mode, etc. I can try stopping random services to find it but that’s not a good way.

**Reply by seerge** [reply]
@Zaat you can just open `%appdata%\GHelper\log.txt` and check which service refuses to stop.

Full list of the actuals service names can be found in the app's code
https://github.com/seerge/g-helper/blob/2c1cb58955d350f1af9b3aa3e1298df033270e76/app/Helpers/OptimizationService.cs#L8
